### ISSUE TITLE
Why two parameters?

### ISSUE THREAD

**Issue by computron** [materialsproject/pymatgen#366]
pymatgen.io.vasp.sets.get_structure_from_prev_run() has both "standardize" and "sym_prec" parameters. The "standardize" parameter seems redundant. i.e. if you set sym_prec = 0 then it would not standardize the cell. If you set standardize=True but sym_prec=0 it will also not standardize the cell. So why have the standardize parameter at all - why not just use sym_prec? Is it for added clarity?

Note that many downstream objects also use two parameters in this way.

**Comment by shyuep** [comment]
This has actually be carefully thought through. You are missing one scenario - where you actually have a sym_prec, e.g., for BS kpoint purposes, but you do **not** want to standardize the cell.

**Comment by computron** [comment]
But in the code I referred to (which is a well defined function) there seems to be no effect of standardize=False and sym_prec>0 (i.e., the sym_prec does nothing).

I also looked at the "from_prev_run()" method in both MPStaticSet and MPNonSCFSet (which both have the two parameters), and both of them also seem to assume that if you have standardize=False, that sym_prec does nothing no matter what you set it to (I didn't test it, just looking at the code). However in these latter two cases, if you were to pass on the "sym_prec" parameter to the "init()" function at the end of the method, it should have the behavior you say - i.e. kpoints use symprec but structure remains the same.
